nav2_simple_navigator/src/simple_navigator.cpp

```diff
@@ -61,9 +68,10 @@ SimpleNavigator::execute(const nav2_tasks::NavigateToPoseCommand::SharedPtr comm
   // localization, while the goal pose is from the incoming command
   auto endpoints = std::make_shared<nav2_tasks::ComputePathToPoseCommand>();
 
+#if 0
```

Comment:
Why is this 'if 0' directive here? Should this code be removed?

Reply:
Shouldn't have been included in the commit. Fixed.